src/BL/CurrecytManager.java

```diff
@@ -0,0 +1,51 @@
+package BL;
+
+import java.util.HashMap;
+
+public class CurrencyManager {
+    //Определяет статическую константу - значение валюты по умолчанию
+    public static final CurrencyTitle defaultCurrency = CurrencyTitle.EUR;
+    // Определяет приватное поле data, предназначенное для хранения экземпляра класса CurrencyExchangeHandler
+    private final CurrencyExchangeHandler data;
+
+    // Конструктор
+    public CurrentManager(String rateFilePath) {
+        this.data = new CurrencyExchangeHandler(rateFilePath);
+    }
+
+    /**
+     * Метод getAllCurrencies - возвращает все валюты и курсы их обмена по отношению к дефолтной валюте.
+     * Использует метод read() класса CurrencyExchangeHandler.
+     * Метод возвращает HashMap с валютами и их курсами.
+    **/
+     public HashMap<String, Double> getAllCurrency () {
```

Comment:
the key of HashMap should be CurrencyTitle type

Reply:
done